### PR TITLE
Add page traversal to transaction history

### PR DESCRIPTION
**Pull request by kymckay**
- Can't use arbitrary page jumping since retrieval API works sequentially, so user can use "next" and "previous" page buttons (or jump to start)
- Selecting sorting complicates the logic and has been left out for now

Part of #134

### CODE REVIEW

**Comment by razzeee** [comment]
This might be easy to improve for anyone that wants to write python code (@danielsilverstone-ct for e.g.) as it seems, that the paging api doesn't have to be cursor based

File: frontend/src/components/payment/transactions/TransactionHistory.tsx [hunk 1]
```diff
@@ -65,10 +93,44 @@ const TransactionHistory: FunctionComponent = () => {
...
+              page * perPage + perPage
+            )}
+          />
+          <div className={styles.controlPage}>
+            <Button
+              type='secondary'
+              onClick={() => setPage(0)}
+              disabled={page === 0}
+            >
+              {t('first-page')}
```
Comment (razzeee) [inline frontend/src/components/payment/transactions/TransactionHistory.tsx:109]: Do we really need first page? Wouldn't expect that to be used often and throws of the balance of the buttons?

**Review by razzeee — APPROVED** [review]
Minor nitpick, seems fine otherwise

**Comment by razzeee** [comment]
We could also use the Material Icons instead of text 
image

(without any additional styling/sizing)

File: frontend/src/components/payment/transactions/TransactionHistory.tsx (hunk 1 above)
Comment (kymckay) [reply frontend/src/components/payment/transactions/TransactionHistory.tsx:109]: Yeah let's rip it out, I was on the fence too

**Comment by kymckay** [comment]
Good shout about the material icons, I like those more, but probably would still want the text to prompt screen readers, etc.

I won't be working tomorrow so if you want to push and merge before Friday feel free to go ahead :smiley:

**Pull request merged**